src/Chapter4.hs

```diff
@@ -628,8 +661,14 @@ Can you implement a monad version of AND, polymorphic over any monad?
 
 🕯 HINT: Use "(>>=)", "pure" and anonymous function
 -}
+andMv1 :: (Monad m) => m Bool -> m Bool -> m Bool
+andMv1 a b = fmap (&&) a <*> b
```

Comment:
If you don't use the `>>=` operator, then the `Monad` constraint is too strong, and it is enough to specify only `Applicative`. However, with `Applicative` you are not able to implement a lazier version of `andM`.
```suggestion
andMv1 :: (Applicative m) => m Bool -> m Bool -> m Bool
andMv1 a b = fmap (&&) a <*> b
```